src/Cli/dotnet/commands/dotnet-workload/update/LocalizableStrings.resx

```diff
@@ -180,4 +180,10 @@
   <data name="WorkloadVersionRequestedNotFound" xml:space="preserve">
     <value>Workload version {0} not found.</value>
   </data>
+  <data name="SpecifiedWorkloadVersionAndSpecificNonWorkloadVersion" xml:space="preserve">
```

Comment:
```suggestion
  <data name="SpecifiedUseWorkloadVersionsWithIncompatibleFeatures" xml:space="preserve">
```

I see what you were going for with the name, but it took me a while to understand what the string meant. This may or may not be more clear.

Reply:
I wouldn't mind a better option for this. My string is long but still not super clear, as you pointed out. I think your string would leave me confused too, though. We can noodle on this a bit and see if one of us comes up with something clear and concise 🙂

Reply:
I think your name is good, maybe I was sleepy